tcms/rpc/tests/test_testcasestatus.py

```diff
@@ -0,0 +1,57 @@
+from tcms.rpc.tests.utils import APITestCase
+from tcms.testcases.models import TestCaseStatus
+from tcms.tests.factories import (
+    ProductFactory,
+    TestCaseFactory,
+    TestPlanFactory,
+    UserFactory,
+    VersionFactory,
+)
+
+
+class TestFilterCaseStatus(APITestCase):
+    """ Tests the XML-RPC testcasestatus.filter method """
```

Comment:
Python is case sensitive so if you are going to document method names use the appropriate case. Also that makes it easier to grep based on the name.